packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -42,6 +42,8 @@ export class Messages {
     `Successfully retrieved zip package from ${zipUrl}.`;
   public static readonly FallingBackToUseLocalTemplateZip =
     "Falling back to use local template zip.";
+  public static readonly ResourceProviderExist = (rp: string): string =>
+    `Resource Provider ${rp} already exist, do not need to create.`;
```

Comment:
exists? don't need to register?

Reply:
Yes, if exist, it will return current RP, won't register a new one.

Reply:
Resolved